docs/developer-docs/setup/cycles/cycles-ledger.md

```diff
@@ -0,0 +1,154 @@
+# Cycles ledger
+
+## Overview
+
+When developing and deploying canisters on the Internet Computer, developers need to have **cycles** to pay for the resources used by their canisters. The **cycles ledger** canister has been developed as an alternative to the **cycles wallet** canister for cycles management. 
+
+The biggest difference between the cycles ledger and cycles wallet is that the cycles ledger is a single global ledger canister, where as the cycles wallet canister must be deployed locally to be used. Additionally, the cycles wallet has some downsides that the cycles ledger remedies. Some of these are:
+
+- Cycles wallets consume cycles themselves, meaning developers need to pay cycles to store cycles.
+- Cycles wallets are a complex solution to associate cycles with principal IDs 
+- Since the cycles wallet canister must be deployed locally, if a cycles wallet canister ID is lost, the cycles in that cycles wallet canister are also lost. 
+
+The cycles ledger solves these complications by providing functionality such as allowing principal IDs to hold cycles, simplifying cycle management, and reducing the risk of loosing cycles. 
+
+The cycles ledger complies with the ICRC-2 standard. In addition to standard ledger functionality, the cycles ledger canister also interacts with the NNS canisters and user canisters to provide cycles ledger-specific functions, such as:
+
+- Depositing cycles to a given principal ID.
```

Comment:
Nit: Depositing cycles is not an action triggered by the cycles ledger. The cycles ledger accepts cycle deposits coming from other canisters.